packages/rocketchat-lib/server/startup/settingsOnLoadCdnPrefix.js

```diff
@@ -0,0 +1,13 @@
+/* globals WebAppInternals*/
+RocketChat.settings.onload('CDN_PREFIX', function(key, value) {
+	if (_.isString(value || false)) {
+		return typeof WebAppInternals !== 'undefined' && WebAppInternals !== null && WebAppInternals.setBundledJsCssPrefix(value);
+	}
+});
+
+Meteor.startup(function() {
+	const value = RocketChat.settings.get('CDN_PREFIX');
+	if (_.isString(value || false)) {
```

Comment:
My wish was testing if the variable is a string or it is not an empty string.

```javascript
"asda"||false - > "asda"
```
```javascript
""||false - > false
```

because ```_.isString("") === true```